tasks/cachebust.js

```diff
@@ -59,9 +60,11 @@ module.exports = function() {
             }));
             var newFilename = addFileHash(file, hash, opts.separator);
 
-            grunt.file.copy(absPath, path.resolve(opts.baseDir, newFilename));
+            if(opts.createCopies) {
+                grunt.file.copy(absPath, path.resolve(opts.baseDir, newFilename));
+            }
 
-            if(opts.deleteOriginals) {
+            if(opts.deleteOriginals && opts.createCopies) {
```

Comment:
I guess so... But then which are the `Originals`? :) `Originals` of hashed files, but we don't hash them.
What I would do is create an three-state option with `rename`, `copy` and `original`, but that wouldn't be backward compatible.
Of course we can go without this line. I will make a commit soon.